### PR TITLE
Admin thumbnail format configuration

### PR DESCRIPTION
**Pull request by core23**
The settings for the admin thumbnail should be configurable.

### CODE REVIEW

**Comment by OskarStark** [comment]
tumbnail -> thumbnail

**Comment by OskarStark** [comment]
could you please add a test?

**Comment by OskarStark** [comment]
can you please rebase this PR; i've fixed the CS problems

**Comment by OskarStark** [comment]
@core23 can you please finish this PR, i would like to merge this, as i need the config :+1: 

thanks

File: DependencyInjection/Configuration.php [hunk 1]
```diff
@@ -34,6 +34,16 @@ public function getConfigTreeBuilder()
             ->children()
                 ->scalarNode('db_driver')->isRequired()->end()
                 ->scalarNode('default_context')->isRequired()->end()
+                ->arrayNode('admin_format')
```
Comment (OskarStark) [inline DependencyInjection/Configuration.php]: maybe `->info()` would be great for what this is used

**Comment by OskarStark** [comment]
ping @core23

**Comment by core23** [comment]
ping @OskarStark

File: DependencyInjection/Configuration.php (hunk 1 above)
Comment (core23) [inline DependencyInjection/Configuration.php]: Done @OskarStark

**Comment by OskarStark** [comment]
@greg0ire could you give us a final review?

File: DependencyInjection/Compiler/AddProviderCompilerPass.php [hunk 2]
```diff
@@ -33,13 +33,7 @@ public function process(ContainerBuilder $container)
...
 
         foreach ($container->findTaggedServiceIds('sonata.media.provider') as $id => $attributes) {
-            $container->getDefinition($id)->addMethodCall('addFormat', array('admin', array(
-                'quality'       => 90,
-                'width'         => 200,
-                'format'        => 'jpg',
-                'height'        => false,
-                'constraint'    => true,
-            )));
+            $container->getDefinition($id)->addMethodCall('addFormat', array('admin', $container->getParameter('sonata.media.admin_format')));
```
Comment (greg0ire) [inline DependencyInjection/Compiler/AddProviderCompilerPass.php]: Please don't use long lines. If there is an horizontal scroll on github, please consider the line too long.

File: DependencyInjection/Configuration.php [hunk 3]
```diff
@@ -34,6 +34,17 @@ public function getConfigTreeBuilder()
...
+                    ->info('Configures the thumbnail preview for the admin')
+                    ->addDefaultsIfNotSet()
+                    ->children()
+                        ->scalarNode('width')->defaultValue(200)->end()
+                        ->scalarNode('height')->defaultValue(false)->end()
```
Comment (greg0ire) [inline DependencyInjection/Configuration.php]: Please add `->info('when set to false, height is computed from width and ratio')` (provided it really is what happens).

Comment (core23) [inline DependencyInjection/Configuration.php]: That's not right. It depends on the resizer.

Comment (greg0ire) [inline DependencyInjection/Configuration.php]: Ok then you can't really provide useful info for this configuration I guess.

**Comment by OskarStark** [comment]
Thank you @core23, good job :+1:

**Pull request merged**